uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/runner/IdentityRunnerArgs.java

```diff
@@ -0,0 +1,35 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.runner;
+
+import com.microsoft.identity.client.ui.automation.installer.AppInstallSource;
+
+/**
+ * Test runner args to be used with {@link IdentityTestRunner}.
+ */
+public class IdentityRunnerArgs {
+
+    public static boolean preferPreInstalledApks = false;
+    public static String brokerSource = AppInstallSource.PlayStore.getName();
```

Comment:
This is still creating global state... I wish we could create another way to do this, but it's less hardcoded than the build-config version.  Suggestion - we should expose setters here and log call information whenever the values are changed.

Reply:
Yeah unfortunately I couldn't find a way to keep these member fields and still be able to access the runner and its members form other places in the code so had to do it this way for now. Sure, I can add setters

Reply:
The only purpose for these setters would be so that we can log any changes as they happen, mind, so our logs can help us figure it out when we make mistakes.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1242/commits/a98f1d9f66221f1d1c48ba60a1f431862e0ed3b1

Reply:
🚢